python/ray/_private/worker.py

```diff
@@ -554,6 +554,18 @@ def debugger_port(self):
         worker_id = self.core_worker.get_worker_id()
         return ray._private.state.get_worker_debugger_port(worker_id)
 
+    @property
+    def get_job_logging_config(self):
+        """Get the job's logging config for this worker"""
+        if not hasattr(self, "core_worker"):
+            return None
+        job_config = self.core_worker.get_job_config()
+        if job_config.serialized_py_logging_config == b"":
```

Comment:
```suggestion
        if not job_config.serialized_py_logging_config:
```

This handles both None and empty string cases

Reply:
what is "empty string" case?

Reply:
This check has already handled the case that when `LoggingConfig` is None.

Reply:
I mean both `serialized_py_logging_config` being None or b""

Reply:
I don't think it will now. It's just that `not` can handle both so more robust.

Reply:
updated: https://github.com/ray-project/ray/pull/47703/commits/8b45732ee95f3964d94225950b5ec1d5d06f1cb9